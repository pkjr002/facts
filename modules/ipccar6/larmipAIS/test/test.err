Traceback (most recent call last):
  File "/scratch/rk509/test.1667487634/ipccar6_postprocess_larmipicesheet.py", line 11, in <module>
    import xarray as xr
ModuleNotFoundError: No module named 'xarray'
