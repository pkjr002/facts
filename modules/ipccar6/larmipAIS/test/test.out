Initiating test in /scratch/rk509/test.1667581577...

Extracting data files...
